spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md

```diff
@@ -424,30 +442,13 @@ function chanUpgradeTry(
         abortTransactionUnless(false)
     }
 
-    // either timeout height or timestamp must be non-zero
-    // if the upgrade feature is implemented on the TRY chain, then a relayer may submit a TRY transaction after the timeout.
-    // this will restore the channel on the executing chain and allow counterparty to use the ChannelUpgradeCancelMsg to restore their channel.
-    if timeoutHeight == 0 && timeoutTimestamp == 0 {
-        restoreChannel()
-        return
-    }
-    
-
     // counterparty-specified timeout must not have exceeded
     if (currentHeight() > timeoutHeight && timeoutHeight != 0) ||
         (currentTimestamp() > timeoutTimestamp && timeoutTimestamp != 0) {
-        restoreChannel()
+        restoreChannel(portIdentifier, channelIdentifier)
         return
     }
 
-    // both channel ends must be mutually compatible.
-    // this function has been left unspecified since it will depend on the specific structure of the new channel.
-    // It is the responsibility of implementations to make sure that the proposed new channels
-    // on either side are correctly constructed according to the new version selected.
-    if !IsCompatible(counterpartyChannel, proposedUpgradeChannel) {
```

Comment:
If I understand correctly, we replaced this with 
```
abortTransactionUnless(counterpartyChannel.ordering != proposedUpgradeChannel.ordering)
```
 to only focus on ordering, right? We should do the same in `chanUpgradeAck`.

Reply:
Good catch 👍